tensorflow/lite/tools/versioning/runtime_version.cc

```diff
@@ -380,6 +380,7 @@ std::string FindMinimumRuntimeVersionForOp(tflite::BuiltinOperator op_code,
            {{BuiltinOperator_MULTINOMIAL, 1}, "2.8.0"},
            {{BuiltinOperator_GELU, 1}, "2.9.0"},
            {{BuiltinOperator_GELU, 2}, "2.9.0"},
+           {{BuiltinOperator_GELU, 3}, "2.11.0"},
```

Comment:
Could you sync to HEAD and update this to the next version (2.12.0) ? Thanks!

Reply:
thanks! I have updated this.